app/controllers/pdl.py

```diff
@@ -11,6 +11,9 @@
 from app.models.allprograms import AllProgram
 
 
+LANDING_PAGE_DATA_PATH = os.path.join("controllers", "data", "landingpage")
+ALLPROGRAM_DATA_JSON = "allprograms.json"
+SUMMARY_DATA_JSON = "summary.json"
 CSP_JSON_DATA_PATH = os.path.join("controllers", "data", "conservation", "csp")
 CSP_MAP_DATA_JSON = "csp_map_data.json"
 CSP_STATE_DISTRIBUTION_DATA_JSON = "csp_state_distribution_data.json"
```

Comment:
Please remove EQIP_JSON_DATA_PATH in line 24. It's a repeated entry, and the path in that line is incorrect. It looks like this change crept in a few weeks ago. I might have missed testing other endpoints as part of that PR review.

Reply:
removed. thanks for the good catch

Reply:
Thank you.